src/python/python.cpp

```diff
@@ -135,7 +135,7 @@ std::string ToFormatDescriptor(ONNXTensorElementDataType type) {
   }
 }
 
-std::unique_ptr<OrtValue> ToTensor(pybind11::array& v) {
+std::unique_ptr<OrtValue> ToOrtValue(pybind11::array& v) {
```

Comment:
One thing we recently discovered about numpy arrays, and we are going to address it, they donot support BFloat16 and any 8-bit types. So users can not use it in Python.

Reply:
Interesting, I messaged you offline about how another team sent us a PR that added 8-bit types. Not BFloat though.

Reply:
Ah, 8-bit floats. It all makes sense now. Yes, we haven't needed those yet. As the data is coming from numpy it should be ok.